bitbots_docs/docs/manual/tutorials/test_robot_hardware.rst

```diff
@@ -0,0 +1,44 @@
+================================================
+Testing the robot hardware and lowlevel software
+================================================
+
+Preliminaries
+-------------
+
+#. Use robot compile to flash correct version on robot
+#. Put robot in a safe spot, on a rope hanging from the ceiling
+#. Check if all cables are correctly connected
+#. Do the test in the provided order, to find out which part is faulty
```

Comment:
_do the test in the provided order_ is not a preliminary but rather a general thing which should be regarded.

I would prefer having that statement as a first sentence under _Test hardware and ros_control_ or something like that